client/components/Zoop.tsx

```diff
@@ -1,14 +1,79 @@
 import React from "react";
-import { useParams } from "react-router-dom";
+import { Fave } from "@prisma/client";
+import FaveButton from "./FaveButton";
 
-const Zoop = () => {
-  const { id } = useParams();
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+import SendIcon from "@mui/icons-material/Send";
+import IconButton from "@mui/material/IconButton";
+import AccountBoxIcon from "@mui/icons-material/AccountBox";
+import Stack from "@mui/material/Stack";
+import AccountCircle from "@mui/icons-material/AccountCircle";
 
-    return(
-        <>
-            <div>This is the page for Zoop # {id}</div>
-        </>
-    )
-}
+type ZoopProps = {
+  zoopId: number;
+  author: string;
+  receiver: string;
+  content: string;
+  faves: Fave[];
+  dateCreated: Date;
+};
 
-export default Zoop;
\ No newline at end of file
+const Zoop = ({
+  zoopId,
+  author,
+  receiver,
+  content,
+  faves,
+  dateCreated,
+}: ZoopProps) => {
+  const date = new Date(dateCreated);
+  const formattedDate = date.toLocaleDateString("en-US", {
```

Comment:
Can you include time as well? Also recommend abbrevating the month name at least

Also maybe add a TODO / Wish list Issue to the project to bring in a library to make these more user friendly (eg. "1 hour ago" or "3 days ago" etc)